README.md

```diff
@@ -643,12 +617,14 @@ Birdcache methods are:
 
 # Authors <a name = "authors"> </a>
 
+- https://github.com/Sjhunt93
```

Comment:
this is ordered by the date you contribute to the python sdk. better be the last one to have some respect

Reply:
Il fix it for you sunshine :)